pkgs/development/python-modules/paddleocr/default.nix

```diff
@@ -0,0 +1,106 @@
+{ lib
+, buildPythonPackage
+, pythonRelaxDepsHook
+, fetchFromGitHub
+, attrdict
+, beautifulsoup4
+, cython
+, fire
+, fonttools
+, lmdb
+, lxml
+, numpy
+, opencv4
+, openpyxl
+, pdf2docx
+, pillow
+, premailer
+, pyclipper
+, pymupdf
+, python-docx
+, rapidfuzz
+, scikit-image
+, shapely
+, tqdm
+, paddlepaddle
+, lanms-neo
+, polygon3
+}:
+
+let
+
+in buildPythonPackage rec {
+  pname = "paddleocr";
+  version = "2.6.1.0";
+
+  src = fetchFromGitHub {
+    owner = "PaddlePaddle";
+    repo = "PaddleOCR";
+    rev = "df54a7b4223c64a8480380f618fe95de3b8cf73b";
+    hash = "sha256-H8sBu6+ApquJy/Kr56cYuAuBEz3cDTPh1QutfyzDnG4=";
+  };
+
+  # TODO: The tests depend, among possibly other things, on `cudatoolkit`.
+  # But Cudatoolkit fails to install.
+  # preCheck = "export HOME=$TMPDIR";
+  # nativeCheckInputs = with pkgs; [ which cudatoolkit ];
+  doCheck = false;
```

Comment:
I think I can just keep `doCheck = false` and change the comment to `# tests require a gpu`?